lambdas/FetchJobsData.py

```diff
@@ -0,0 +1,62 @@
+import psycopg2
+import json
+import os
+import logging
+
+try:
+    # for local development, load environment variables from .env file
+    import dotenv
+    dotenv.load_dotenv()
+except ImportError:
+    logging.warning('dotenv not available, assuming running in AWS environment')
+
+def get_db_connection():
+    try:
+        return psycopg2.connect(
+            host=os.environ['DB_HOST'],
+            port=os.environ['DB_PORT'],
+            dbname=os.environ['DB_NAME'],
+            user=os.environ['DB_USER'],
+            password=os.environ['DB_PASSWORD'])
+    except Exception as e:
+        logging.error(f"Error connecting to the database: {e}")
```

Comment:
For example here, instead of logging this, you can log:
```
{
    instance: db, // tells you where it fails
    action: connect, // tells you what it does
    message: e.message, // message of the error
    tracestack: e
    ... // more here
}